gradle/libs.versions.toml

```diff
@@ -22,14 +23,14 @@ evo = { module = "org.atteo:evo-inflector", version.ref = "evo" }
 
 test-kotlinx-coroutines = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-test", version.ref = "kotlinx-coroutines" }
 test-sqlite = { module = "org.xerial:sqlite-jdbc", version.ref = "sqlite" }
-test-log4j = { module = "org.apache.logging.log4j:log4j-core", version.ref = "log4j" }
-test-log4j-impl = { module = "org.apache.logging.log4j:log4j-slf4j2-impl", version.ref = "log4j" }
+test-h2 = { module = "com.h2database:h2", version.ref = "h2" }
+test-logback = { module = "ch.qos.logback:logback-classic", version.ref = "logback" }
```

Comment:
why're we prefixing with `test-`, just let it be `h2`, `logback` etc. and we import as a test impl in our gradle.

Reply:
This is to prevent accidental usage of these dependencies in non-test environments by providing them a label